portfolio/src/main/webapp/comments-script.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches the comments input and builds the UI into a list.
+ */
+
+function getComments() {
+    fetch('/comments').then(response => response.json()).then((comments) => {
+        const commentsElement = document.getElementById('comments-container');
```

Comment:
Prefer this to be its own function. Any time a lambda is more than one line I like to make it it's own method.